modules/accredited_representative_portal/app/controllers/accredited_representative_portal/v0/decisions_controller.rb

```diff
@@ -0,0 +1,69 @@
+# frozen_string_literal: true
+
+module AccreditedRepresentativePortal
+    module V0
+        class DecisionsController < ApplicationController
+            before_action :set_power_of_attorney_request, only: [:show, :create]
+
+            def create
+                if @power_of_attorney_request.resolution.present?
+                    render json: { error: 'Resolution already exists' }, status: :unprocessable_entity
+                    return
+                end
+        
+                creator_id = @current_user.id
+        
+                type = power_of_attorney_request_params[:declination_reason].present? ? PowerOfAttorneyRequestDecision::Types::DECLINATION :  PowerOfAttorneyRequestDecision::Types::ACCEPTANCE
```

Comment:
params here should come with a type e.g.:
```json
{
  "decision_type": "declination",
  "reason": "Missing authorizations"
}
```

Checking existing serialization outbound from the app should suggest a uniform way of representing our objects in different interactions.